src/steps/trackReplaceAroundStep.ts

```diff
@@ -79,6 +80,19 @@ export function trackReplaceAroundStep(
     attrs,
     slice
   )
+
+  let fragment = setFragmentAsInserted(
+    newSliceContent,
+    trackUtils.createNewInsertAttrs(attrs),
+    oldState.schema
+  )
+
+  const isWrapStep = step.from === step.gapFrom && step.to === step.gapTo && newSliceContent.size === 0;
```

Comment:
lets create a function and move it to utils as we did with isSplitStep so it's more in alignment with how we organise the work with other steps.